packages/trader/src/Modules/Reports/Components/reports-meta.jsx

```diff
@@ -1,10 +1,18 @@
 import classNames from 'classnames';
 import React from 'react';
 
-const ReportsMeta = ({ filter_component, className, optional_component }) => (
+const ReportsMeta = ({ filter_component, className, optional_component, is_statement }) => (
     <div className={classNames('reports__meta', className)}>
         {optional_component}
-        {filter_component && <div className='reports__meta-filter'>{filter_component}</div>}
+        {filter_component && (
+            <div
+                className={classNames('reports__meta-filter', {
+                    'reports__meta-filter-statement': is_statement,
```

Comment:
Since this is a modifier `className`, suggest to change it to below:

```suggestion
                    'reports__meta-filter--statement': is_statement,
```